addons/zeusFactory/functions/fnc_getTransportType.sqf

```diff
@@ -0,0 +1,36 @@
+// getTransportType
+#include "script_component.hpp"
+
+params ["_factoryLogic", "_transportType", "_side"];
+TRACE_3("getTransportType",_factoryLogic,_transportType,_side);
+
+private _vehType = "";
+switch (_transportType) do {
+case TRANSPORT_CAR_RTB;
+case TRANSPORT_CAR_FOLLOW: {_vehType = _factoryLogic getVariable [QGVAR(carType), ""]};
+case TRANSPORT_APC_RTB;
+case TRANSPORT_APC_FOLLOW: {_vehType = _factoryLogic getVariable [QGVAR(apcType), ""]};
+case TRANSPORT_HELI;
+case TRANSPORT_PARADROP: {_vehType = _factoryLogic getVariable [QGVAR(heliType), ""]};
+};
+
+private _crewType = ["_crew_F", "_Helipilot_F"] select (_vehType isKindOf "Air");
+_crewType = switch (_side) do {
+case (west): {"B" + _crewType};
+case (east): {"O" + _crewType};
+case (resistance): {"I" + _crewType};
+};
```

Comment:
We have this pre-fixing logic in a few of our addons, is there a good way to commonize it?